substrate/client/network/sync/src/lib.rs

```diff
@@ -2618,12 +2618,25 @@ mod test {
 
 	/// Build and import a new best block.
 	fn build_block(client: &mut Arc<TestClient>, at: Option<Hash>, fork: bool) -> Block {
+		if fork {
+			build_block_with_fork_data(client, at, Some((vec![1, 2, 3], vec![4, 5, 6])))
+		} else {
+			build_block_with_fork_data(client, at, None)
+		}
+	}
+
+	/// Build and import a new best block and the fork data.
+	fn build_block_with_fork_data(
+		client: &mut Arc<TestClient>,
+		at: Option<Hash>,
+		fork_data: Option<(Vec<u8>, Vec<u8>)>,
+	) -> Block {
```

Comment:
Why is `build_block()` with `fork: true` not enough? There are only two forks in the test at the same block number, so it should be enough to create one of them with `fork: false` and another with `fork: true`?

Reply:
yeah not needed, removed it